config_examples/config_full.example.json

```diff
@@ -172,7 +172,18 @@
         "jwt_secret_key": "somethingrandom",
         "CORS_origins": [],
         "username": "freqtrader",
-        "password": "SuperSecurePassword"
+        "password": "SuperSecurePassword",
+        "ws_token": "a_secret_ws_token",
+        "enable_message_ws": false
+    },
+    "external_message_consumer": {
+        "enabled": false,
+        "producers": [
+          {
```

Comment:
can we add a `name` to each producer so we can filter to it when getting the external dataframe?

Reply:
:+1: